services/src/Microsoft.Azure.IIoT.Services.OpcUa.Onboarding/src/Program.cs

```diff
@@ -109,6 +111,9 @@ public class Program {
 
             // register logger
             builder.RegisterLogger(LogEx.ApplicationInsights(configuration));
```

Comment:
Wouldn't it make sense to have a consistent configuration for both logger and MetricsLogger?